admin/c2cgeoportal_admin/views/layertree.py

```diff
@@ -34,19 +32,7 @@ def index(self):
     @view_config(route_name='layertree_nodes',
                  renderer='json')
     def nodes(self):
-        # Preload treeitems and treegroups children relations.
-        # Note that session.identity_map only keep weak references
-        # so we need to keep strong references until the end nodes rendering.
-        groups = self._dbsession.query(TreeGroup). \
-            options(subqueryload('children_relation')). \
-            all()
-
-        items = self._dbsession.query(TreeItem). \
-            all()
-
-        themes = self._dbsession.query(Theme). \
-            order_by(Theme.ordering). \
-            all()
```

Comment:
This is for performance, not functional, it reduce the tree loading by 10 or more